.github/workflows/pull_request.yml

```diff
@@ -39,6 +39,14 @@ jobs:
       - name: Gradle compile JMH benchmarks
         run: ./gradlew compileJmh --info
 
+  macos-tests:
+    name: macOS tests
+    # Workaround https://github.com/nektos/act/issues/1875
+    uses: swiftlang/swift-java/.github/workflows/macos_tests.yml@wip-macos-ci
+    with:
+      runner_pool: general
```

Comment:
We don't have `general` label in swiftlang.

Reply:
Ah thank you!